README.md

```diff
@@ -326,3 +326,38 @@ that identifies the clone and *clone_seq* its nucleotide sequence.
 A [loom file](http://linnarssonlab.org/loompy/).
 
 This file is created only if option `--loom` (or `-l`) is used.
+
+
+# Creating a quality control report
+
+```shell
+trex qc_report --plot-jaccard-matrix --plot-hamming-distance DIRECTORY
```

Comment:
I wonder: Wouldn’t it be nicer if the name for the subcommand were just `qc`?

Reply:
Wouldn't it be too short to understand? Is """quality_control""" too long?

Reply:
Isn’t QC a more or less standard abbreviation? (This is something that needs to be typed in, so the argument about "code is more often read than written" doesn’t apply.)

Reply:
I agree. As the people who decided on mkdir, rm, ls, will probably agree. Expect updates...